package.json

```diff
@@ -2,53 +2,108 @@
   },
-  "homepage": "https://github.com/WebDevStudios/wd_s#readme"
+  "homepage": "https://github.com/WebDevStudios/wd_s#readme",
+  "main": "src/js/app.js",
+  "dependencies": {
+    "slick-carousel": "^1.8.1"
+  },
+  "devDependencies": {
+    "@webdevstudios/js-coding-standards": "^1.0.0-beta1",
+    "@wordpress/eslint-plugin": "^4.0.0",
+    "autoprefixer": "^9.7.4",
+    "browser-sync": "^2.26.7",
+    "cssnano": "^4.1.10",
+    "eslint": "^6.8.0",
+    "imagemin-cli": "^5.1.0",
+    "mkdirp": "^1.0.3",
+    "node-sass": "^4.13.1",
+    "npm-run-all": "^4.1.5",
+    "onchange": "^6.1.0",
+    "postcss-cli": "^7.1.0",
+    "prettier": "^1.19.1",
+    "rimraf": "^3.0.2",
+    "stylelint": "^13.1.0",
+    "stylelint-config-wordpress": "^16.0.0",
+    "stylelint-order": "^4.0.0",
+    "stylelint-scss": "^3.14.2",
+    "svgo": "^1.3.2",
+    "svgstore-cli": "^1.3.1",
+    "uglify-es": "^3.3.9",
+    "wp-pot-cli": "^1.3.0"
+  },
+  "scripts": {
+    "clean": "rimraf dist/{css/*,js/*,images/*,languages/*}",
+    "optimizecss": "postcss dist/css/style.css -c postcss.config.js -o dist/css/style.css",
+    "scss": "node-sass src/scss/style.scss -o dist/css",
+    "gutenberg": "node-sass src/scss/gutenberg-blocks-style.scss -o dist/css",
+    "lint": "eslint src/js || true",
+    "lint-scss": "stylelint src/scss/**/*.scss --syntax scss --config .stylelintrc || true",
+    "uglify": "mkdirp dist/js -p && uglifyjs src/js/concat/*.js -m -o dist/js/app.js && uglifyjs src/js/concat/*.js -m -c -o dist/js/app.min.js",
+    "customizer": "uglifyjs src/js/customizer.js -m -o dist/js/customizer.js && uglifyjs src/js/customizer.js -m -c -o dist/js/customizer.min.js",
+    "polyfill": "uglifyjs src/js/babel-polyfill.min.js -m -o dist/js/babel-polyfill.js && uglifyjs src/js/babel-polyfill.min.js -m -c -o dist/js/babel-polyfill.min.js",
+    "imagemin": "imagemin src/images/*.* --out-dir=dist/images/",
+    "icons": "svgo -f src/images/icons -o dist/images/icons && svgstore dist/images/icons/*.svg --inline -o dist/images/icons/sprite.svg",
+    "pot": "mkdirp dist/languages && wp-pot --src './*.php' --dest-file 'dist/languages/_s.pot'",
+    "php": "phpcs -p -s -n . --standard=phpcs.xml.dist --extensions=php || true || exit",
+    "serve": "browser-sync start --https --proxy 'https://testing.test' --no-open --files \"dist/css/*.css, dist/js/*.js, **/*.html, **/*.php, !node_modules/**/*.html\"",
+    "build:css": "run-s lint-scss scss gutenberg optimizecss",
+    "build:js": "run-s lint uglify polyfill customizer",
+    "build:images": "run-s imagemin icons",
+    "build:pot": "run-s pot",
+    "build": "run-s clean build:*",
+    "watch:css": "onchange \"src/scss\" -- run-s build:css",
+    "watch:js": "onchange \"src/js\" -- run-s build:js",
+    "watch:images": "onchange \"src/images\" -- run-s build:images",
+    "watch:php": "onchange './*.php', './**/*.php' -- run-s php",
+    "watch": "run-p serve watch:*",
+    "postinstall": "run-s build"
```

Comment:
For many of these you can enforce local version by using, e.g. `"clean": "rimraf dist/{css/*,js/*,images/*,languages/*}",` could be `"clean": "./node_modules/.bin/rimraf dist/{css/*,js/*,images/*,languages/*}",`. If the user has a global version installed, it may cause conflicts for `npm`. All the binaries you use here should be in `./node_modules/.bin/*`.